docs/src/tutorial/clibraries.rst

```diff
@@ -201,23 +256,53 @@ if other applications also use C-Alg.
 Static Linking
 ---------------
 
-To build the c-code automatically we need to include compiler directives in `queue.pyx`::
+To build the c-code automatically we need to include compiler directives in `queue.pyx`
 
-    # distutils: sources = c-algorithms/src/queue.c
-    # distutils: include_dirs = c-algorithms/src/
+.. tabs::
 
-    cimport cqueue
+    .. group-tab:: Pure Python
 
-    cdef class Queue:
-        cdef cqueue.Queue* _c_queue
-        def __cinit__(self):
-            self._c_queue = cqueue.queue_new()
-            if self._c_queue is NULL:
-                raise MemoryError()
+        .. code-block:: python
 
-        def __dealloc__(self):
-            if self._c_queue is not NULL:
-                cqueue.queue_free(self._c_queue)
+            # distutils: sources = c-algorithms/src/queue.c
+            # distutils: include_dirs = c-algorithms/src/
+
+            from cython.cimports import cqueue
```

Comment:
i think those tabs should go to separate files

Reply:
and we should add cython import here, because of `# distutils`

Reply:
`import cython` added.